assets/home/sections/quick-links.js

```diff
@@ -2,21 +2,135 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Icon, external } from '@wordpress/icons';
+import { useState, useRef } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Section from '../section';
+import { Grid, Col } from '../grid';
+
+const quickLinksSpecialMapping = {
+	'sensei://install-demo-course': ( { setTitle, originalTitle, timer } ) => {
+		setTitle( 'Installing' );
+		const installedMessage = 'Installed!';
+		const clear = () => {
+			if ( timer.current ) {
+				clearTimeout( timer.current );
+			}
+			timer.current = 0;
+		};
+		const run = () => {
+			setTitle( ( oldTitle ) => {
+				if ( installedMessage === oldTitle ) {
+					clear();
+					return originalTitle;
+				}
+				if ( ! oldTitle.includes( '...' ) ) {
+					timer.current = setTimeout( run, 500 );
+					return oldTitle + '.';
+				}
+				timer.current = setTimeout( run, 2000 );
+				return installedMessage;
+			} );
+		};
+		clear();
+		timer.current = setTimeout( run, 500 );
+	},
+};
 
 /**
- * Quick Links section component.
+ * Component that shows a Quick Link.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.item The item to show.
+ */
+const QuickLink = ( { item } ) => {
+	let { url, title: originalTitle } = item;
+	const [ title, setTitle ] = useState( originalTitle );
+	const timer = useRef( 0 );
+
+	let onClick = null;
+
+	const callback = quickLinksSpecialMapping[ url ];
+	if ( !! callback ) {
+		url = '#';
+		onClick = ( e ) => {
+			e.preventDefault();
+			callback( { setTitle, originalTitle, timer } );
+		};
+	}
+	const isExternal = url && url !== '#' && ! url.includes( '/wp-admin/' );
+	return (
+		<li>
+			{ /* eslint-disable-next-line react/jsx-no-target-blank */ }
```

Comment:
I think instead of ignoring this rule, we could just add the `rel="noreferrer"` to the link. Or you didn't want it on purpose?

Reply:
I didn't add it just because most of the external links seem to point to senseilms.com, so I wasn't sure if we should add the `rel="noreferrer"` or not.

We can, definitely, add the task, but I was a little in doubt while developing the feature :P

Reply:
Now that I'm using the Link component, as added here: [0c0b7fe](https://github.com/Automattic/sensei/pull/5896/commits/0c0b7fed21e3c6f274078a3bcd481c2b7f69cdb9) - and that Link component adds `rel="noreferrer"`, I'm going to make this conversation as resolved. :)